simulation/traffic_simulator/include/traffic_simulator/data_type/speed_change.hpp

```diff
@@ -24,17 +24,17 @@ namespace traffic_simulator
 namespace speed_change
 {
 enum class Transition {
-  // @todo CUBIC,
+  // @todo Add CUBIC transition type.
```

Comment:
enum member is not type.
```suggestion
  // @todo Add CUBIC.
```